### ISSUE TITLE
When my data volume is too large, the igure displayed by the browser is blank

### ISSUE THREAD

**Issue by Zxlan** [plotly/plotly.py#3734]
Thanks for your interest in Plotly.py!

Before opening an issue, please search for existing and closed issues :)

**Please** accompany bug reports with a reproducible example. Please use the [latest version](https://github.com/plotly/plotly.py/blob/master/CHANGELOG.md) of plotly.py in your report unless not applicable.

Note that GitHub Issues are meant to be used for bug reports and feature requests only. Implementation or usage questions should be asked on [community.plotly.com](https://community.plotly.com/c/graphing-libraries/python/10) or on Stack Overflow (tagged [`plotly`](https://stackoverflow.com/questions/tagged/plotly)).

**Comment by Zxlan** [comment]
**My code ：**
pointclouds = pointclouds.detach().cpu().subsample(max_points_per_pointcloud)
    verts = pointclouds.points_packed()
    features = pointclouds.features_packed()

    color = None
    if features is not None:
        if features.shape[1] == 4:  # rgba
            template = "rgb(%d, %d, %d, %f)"
            rgb = (features[:, :3].clamp(0.0, 1.0) * 255).int()
            color = [template % (*rgb_, a_) for rgb_, a_ in zip(rgb, features[:, 3])]

        if features.shape[1] == 3:
            template = "rgb(%d, %d, %d)"
            rgb = (features.clamp(0.0, 1.0) * 255).int()
            color = [template % (r, g, b) for r, g, b in rgb]

    row = subplot_idx // ncols + 1
    col = subplot_idx % ncols + 1
    fig.add_trace(
        go.Scatter3d(
            x=verts[:, 0],
            y=verts[:, 1],
            z=verts[:, 2],
            marker={"color": color, "size": marker_size},
            mode="markers",
            name=trace_name,
        ),
        row=row,
        col=col,
    )
**there are more than 300,000 points in pointclouds**

**Comment by WXY-Belief** [comment]
same problem ,did you solve this problem ,if you solve contract me ,thanks a lot.

**Comment by Zxlan** [comment]
@WXY-Belief I researched the code and found that plotly relies on JavaScript, and browsers have memory constraints on JavaScript. You can try to use a browser with no memory limit to display, it may be useful
